repository/src/main/resources/db/migration/V0002__RDM-10523.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS event_case_field_event_id_case_field_id_idx ON public.event_case_field USING btree (event_id, case_field_id);
```

Comment:
file name convention is now decided. pls check channels for it.

Reply:
Renamed the file.